src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -212,10 +224,26 @@ public void TestSuggestEmptyEnumList(string expression, params string[] expected
         public void TestSuggestEmptyAll(string expression, params string[] expectedSuggestions)
         {
             Preview.FeatureFlags.StringInterpolation = true;
-            var actualSuggestions = SuggestStrings(expression, MinimalEnums);
+            var config = MinimalEnums;
+            var actualSuggestions = SuggestStrings(expression, config);
+            Assert.Equal(expectedSuggestions, actualSuggestions);
+
+            // With adjusted config 
+            AdjustConfig(config);
+            actualSuggestions = SuggestStrings(expression, config);
             Assert.Equal(expectedSuggestions, actualSuggestions);
         }
 
+        // Add an extra (empy) symbol table into the config and ensure we get the same results. 
+        private void AdjustConfig(PowerFxConfig config)
+        {
+            config.SymbolTable = new SymbolTable 
```

Comment:
>SymbolTable [](http://example.com/codeflow?start=37&length=11)

This new symbol table has a null EnumStoreBuilder 
Hence, causing config.EnumStoreBuilder to return null and hit the nullref. 

Fix is to avoid calling config.EnumStoreBuilder all together, and instead call config.EnumStore  - and that enum store can properly enumerate from the full chain.